### PR TITLE
FEATURE: SD-328 add parent attr to drill message

### PR DESCRIPTION
**Pull request by naibinh**
## PR Checklists
- [x] Squash commits
- [x] Successful unit test
- [x] Successful storybook test
- [x] Successful AD testcafe
- [x] Successful KD testcafe
---
## Related PRs
AD: https://github.com/gooddata/gdc-analytical-designer/pull/2102
KD: https://github.com/gooddata/gdc-dashboards/pull/1753

### CODE REVIEW

**Comment by naibinh** [comment]
extended test - storybook

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-react-components-opensource-repository-check/3625/ : SUCCESS in 19s
- https://checklist.intgdc.com/job/client-libs/job/gooddata-react-components-unit-tests-zuul-docker-el7/2899/ : SUCCESS in 4m 43s

**Comment by yenkins** [comment]
Link to Storybook test result:

https://checklist.intgdc.com/job/client-libs/job/gooddata-react-components-storybook-zuul-docker/1549/artifact/test-storybook/html-report/index.html

**Comment by gdgate** [comment]
Build failed (check-extended pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-react-components-opensource-repository-check/3626/ : SUCCESS in 23s
- https://checklist.intgdc.com/job/client-libs/job/gooddata-react-components-pull-request-dispatcher/949/ : UNSTABLE in 19m 51s

**Comment by naibinh** [comment]
extended test - storybook

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-react-components-opensource-repository-check/3637/ : SUCCESS in 7s
- https://checklist.intgdc.com/job/client-libs/job/gooddata-react-components-unit-tests-zuul-docker-el7/2908/ : SUCCESS in 2m 43s

**Comment by yenkins** [comment]
Link to Storybook test result:

https://checklist.intgdc.com/job/client-libs/job/gooddata-react-components-storybook-zuul-docker/1553/artifact/test-storybook/html-report/index.html

**Comment by gdgate** [comment]
Build succeeded (check-extended pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-react-components-opensource-repository-check/3638/ : SUCCESS in 18s
- https://checklist.intgdc.com/job/client-libs/job/gooddata-react-components-pull-request-dispatcher/953/ : SUCCESS in 16m 31s

**Comment by naibinh** [comment]
extended test - storybook

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-react-components-opensource-repository-check/3649/ : SUCCESS in 7s
- https://checklist.intgdc.com/job/client-libs/job/gooddata-react-components-unit-tests-zuul-docker-el7/2914/ : SUCCESS in 7m 37s

**Comment by yenkins** [comment]
Link to Storybook test result:

https://checklist.intgdc.com/job/client-libs/job/gooddata-react-components-storybook-zuul-docker/1560/artifact/test-storybook/html-report/index.html

**Comment by gdgate** [comment]
Build failed (check-extended pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-react-components-opensource-repository-check/3650/ : SUCCESS in 16s
- https://checklist.intgdc.com/job/client-libs/job/gooddata-react-components-pull-request-dispatcher/959/ : UNSTABLE in 13m 05s

**Comment by naibinh** [comment]
extended test - storybook

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-react-components-opensource-repository-check/3651/ : SUCCESS in 6s
- https://checklist.intgdc.com/job/client-libs/job/gooddata-react-components-unit-tests-zuul-docker-el7/2915/ : SUCCESS in 4m 27s

**Comment by yenkins** [comment]
Link to Storybook test result:

https://checklist.intgdc.com/job/client-libs/job/gooddata-react-components-storybook-zuul-docker/1561/artifact/test-storybook/html-report/index.html

**Comment by gdgate** [comment]
Build succeeded (check-extended pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-react-components-opensource-repository-check/3652/ : SUCCESS in 8s
- https://checklist.intgdc.com/job/client-libs/job/gooddata-react-components-pull-request-dispatcher/960/ : SUCCESS in 19m 52s

File: src/components/visualizations/chart/test/chartOptionsBuilder.spec.ts [hunk 1]
```diff
@@ -3553,4 +3555,182 @@ describe('chartOptionsBuilder', () => {
...
+            const parentAttribute = getParentAttribute(Array(2).fill({
+                attributeHeaderItem: {
+                    name: 'East Coast',
+                    uri: '/gdc/md/test/obj/21/elements?id=1'
+                }
+            }));
+            const result = getDrillableSeriesWithParentAttribute([series], parentAttribute)[0];
+            const [ firstItem, secondItem ] = result.data;
+
+            expect(firstItem).toEqual({
```
Comment (ZdenekVidensky) [inline src/components/visualizations/chart/test/chartOptionsBuilder.spec.ts]: Can you please unify these two expect into one expect?

File: src/components/visualizations/chart/test/chartOptionsBuilder.spec.ts [hunk 2]
```diff
@@ -3553,4 +3555,182 @@ describe('chartOptionsBuilder', () => {
...
```
Comment (ZdenekVidensky) [inline src/components/visualizations/chart/test/chartOptionsBuilder.spec.ts]: Same as previous

Comment (naibinh) [reply src/components/visualizations/chart/test/chartOptionsBuilder.spec.ts]: Fixed.

File: src/components/visualizations/chart/test/chartOptionsBuilder.spec.ts (hunk 1 above)
Comment (naibinh) [reply src/components/visualizations/chart/test/chartOptionsBuilder.spec.ts]: Fixed.

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-react-components-opensource-repository-check/3699/ : SUCCESS in 14s
- https://checklist.intgdc.com/job/client-libs/job/gooddata-react-components-unit-tests-zuul-docker-el7/2951/ : SUCCESS in 7m 26s

**Comment by gdgate** [comment]
Build in pipeline check aborted.

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-react-components-opensource-repository-check/3717/ : SUCCESS in 59s
- https://checklist.intgdc.com/job/client-libs/job/gooddata-react-components-unit-tests-zuul-docker-el7/2968/ : SUCCESS in 17m 47s

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-react-components-opensource-repository-check/3728/ : SUCCESS in 7s
- https://checklist.intgdc.com/job/client-libs/job/gooddata-react-components-unit-tests-zuul-docker-el7/2977/ : SUCCESS in 6m 35s

**Comment by gdgate** [comment]
Build in pipeline check aborted.

**Comment by gdgate** [comment]
Build succeeded (gate pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-react-components-merge-compliance-check/809/ : SUCCESS in 27s
- https://checklist.intgdc.com/job/client-libs/job/gooddata-react-components-unit-tests-zuul-docker-el7/2980/ : SUCCESS in 2m 38s
- https://checklist.intgdc.com/job/client-libs/job/gooddata-react-components-storybook-zuul-docker/1592/ : SUCCESS in 19m 40s
- https://checklist.intgdc.com/job/client-libs/job/gooddata-react-components-examples-zuul-docker/806/ : SUCCESS in 29m 21s

**Pull request merged**